linux_os/guide/services/ntp/chronyd_specify_remote_server/bash/shared.sh

```diff
@@ -0,0 +1,9 @@
+# platform = Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8,Red Hat Virtualization 4,multi_platform_fedora,multi_platform_ol
```

Comment:
Please be consistent about applicable platforms. The corresponding rule.yml specifies different prodtype. Change it to be aligned.

Reply:
@vojtapolasek Please, align the platforms